app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h2>Contestants</h2>
+<% @contestants.each do |c| %>
+<div id="<%=c.id%>">
+  <h3><%= c.name %></h3>
+  <p>Projects: <% c.find_contestant_projects.each do |p| %>
```

Comment:
This is more of a nit, but the user story displays a comma separated list, so to be fully loyal to that user story's spec, you could instead do `contestant.find_contestant_projects.join(',')` to create that list rather than iterating

Also, you don't need to use your model method here, as you can call the association directly on the contestant: `c.projects.join(", ")`